File: 2.Traverse/Python/11.EvenArraySum.py
def even_array_sum(arr, N):
    sum = 0
    for i in range(N):
        if arr[i] % 2 == 0:
            sum += arr[i]
    return sum

# Hardcoded input
N = 5
arr = [1, 2, 3, 4, 5]
res = even_array_sum(arr, N)
print(res)

File: 2.Traverse/Javascript/11.EvenArraySum.js
function Even_Array_Sum(arr,N){
    
    let sum = 0
    
    for(let i=0;i<N;i++){
        if(arr[i] % 2==0){
            sum += arr[i]
        }
    }
    
    return sum
}




let N = 5
let arr = [1,2,3,4,5]
let res = Even_Array_Sum(arr,N)
console.log(res)